wradlib/georef/raster.py

```diff
@@ -256,7 +256,9 @@ def create_raster_dataset(data, coords, projection=None, nodata=-9999):
 
     # initialize geotransform
     x_ps, y_ps = coords[1, 1] - coords[0, 0]
-    geotran = [coords[0, 0, 0], x_ps, 0, coords[0, 0, 1], 0, y_ps]
+    upper_corner_x = coords[0, 0, 0] - x_ps/2
```

Comment:
This is only true, if we have "center" coordinates. The function was implemented by using the UPPER LEFT pixel corner as this is also the anchor point for the gdal geotransform.

Reply:
Yes I know you used the edge coordinates.
This is just the center coordinates version, nothing wrong with your version.